### ISSUE TITLE
Average Stats always empty

### ISSUE THREAD

**Issue by benschlegel** [anthok/overwatch-api#16]
Im trying to get the average stats per 10 minutes, aka the `average_stats` field in the output, but whatever battletag im using, this is the only field thats always empty (its empty in every region too, not only the one that the player plays in)

**Comment by Drummersbrother** [comment]
I have been able to get non-empty `average_stats` fields. See the following:
```python
import asyncio

from overwatch_api.core import AsyncOWAPI
from overwatch_api.constants import *

async def testing(loop):
    # Instantiating the api
    client = AsyncOWAPI()

    data = {}

    print('Testing......[get_hero_stats]')
    print((await client.get_hero_stats("Danielfrogs#2552", platform=PC))
          ['eu']['stats']['quickplay']['symmetra']['average_stats'])

if __name__ == "__main__":
    loop = asyncio.get_event_loop()
    loop.run_until_complete(testing(loop))
```
When I run it, I get
```none
Testing......[get_hero_stats]
{'teleporter_uptime_average': 0.16}
```
I don't know why almost all heroes have empty `average_stats` fields (Only Zarya and Symmetra have any data for `Danielfrogs#2552`), but that field is at least *not always* empty.

**Comment by benschlegel** [comment]
this still isnt really getting the per 10 minutes stat, just some generals and that only for 2 heros

**Comment by Drummersbrother** [comment]
Hmm 🤔, maybe OWAPI isn't responding with updated data.

**Comment by benschlegel** [comment]
Any idea what i could do to get those stats tho?

**Comment by Drummersbrother** [comment]
No, I unfortunately don't know how or where to get those stats 😕.

**Comment by benschlegel** [comment]
Any way i could reach out to the developer so he could fix it?

**Comment by Drummersbrother** [comment]
Once you find even more examples of battletags lacking average stats, you could just shoot the developer(s?) of OWAPI an email about it. But make sure to really confirm this problem, I don’t want to bother the dev(s) without a reason to 👍.
